client/components/tinymce/plugins/wpcom-help/help-modal.jsx

```diff
@@ -105,36 +124,49 @@ class HelpModal extends React.Component {
 	};
 
 	render() {
+		const translate = this.props.translate;
 		const defaultText = this.props.macosx
-			? this.props.translate( 'Default shortcuts, Command + key:', { context: 'Mac shortcuts' } )
-			: this.props.translate( 'Default shortcuts, Ctrl + key:', { context: 'Windows shortcuts' } );
+			? translate( 'Default shortcuts, Command + key:', { context: 'Mac shortcuts' } )
+			: translate( 'Default shortcuts, Ctrl + key:', { context: 'Windows shortcuts' } );
 
 		const additionalText = this.props.macosx
-			? this.props.translate( 'Additional shortcuts, Control + Option + key:', {
+			? translate( 'Additional shortcuts, Control + Option + key:', {
 					context: 'Mac shortcuts',
 				} )
-			: this.props.translate( 'Additional shortcuts, Shift + Alt + key:', {
+			: translate( 'Additional shortcuts, Shift + Alt + key:', {
 					context: 'Windows shortcuts',
 				} );
 
+		const tableHead = this.getTableHead();
+
 		return (
 			<Dialog
 				isVisible={ this.props.showDialog }
 				buttons={ this.getButtons() }
 				additionalClassNames="wpcom-help__dialog"
 				onClose={ this.props.onClose }
 			>
-				<h2 className="wpcom-help__heading">{ this.props.translate( 'Keyboard Shortcuts' ) }</h2>
+				<h2 className="wpcom-help__heading">{ translate( 'Keyboard Shortcuts' ) }</h2>
 				<p>{ defaultText }</p>
 				<table className="wpcom-help__table">
-					{ this.getTableHead() }
+					{ tableHead }
 					<tbody>{ this.defaultShortcuts().map( this.renderRow, this ) }</tbody>
 				</table>
 				<p>{ additionalText }</p>
 				<table className="wpcom-help__table">
-					{ this.getTableHead() }
+					{ tableHead }
 					<tbody>{ this.additionalShortcuts().map( this.renderRow, this ) }</tbody>
 				</table>
+				<p>{ translate( 'Formatting shortcuts, key, then space:' ) }</p>
```

Comment:
Could we try to improve this text (this will also make it easier to translate)? For example: `Formatting shortcuts: enter the text, then press Space:` Alternatively `Formatting shortcuts: enter the snippet, then press Space:`.

This of course also requires changing the `tableHead` appropriately, but after all these are not keys but text that needs to be entered.